app/src/test/java/com/x8bit/bitwarden/data/auth/manager/UserLogoutManagerTest.kt

```diff
@@ -157,6 +170,41 @@ class UserLogoutManagerTest {
         }
     }
 
+    @Suppress("MaxLineLength")
+    @Test
+    fun `softLogout should switch active user but keep previous user in accounts list`() {
+        val userId = USER_ID_1
+        val vaultTimeoutInMinutes = 360
+        val vaultTimeoutAction = VaultTimeoutAction.LOGOUT
+
+        mockkStatic(Toast::class)
```

Comment:
Don't forget to unmockk, or wrap everything in a closure so it gets unmockked automagically.

Reply:
oh good call!

Reply:
oh this exists

```
@AfterEach
    fun tearDown() {
        unmockkStatic(Toast::class)
    }
``` 
on line 75 so we good

Reply:
We always call `unmockkStatic` in the `tearDown`, this ensures that if a test were to throw an exception, the other tests will not be affected by the `mockkStatic`.

Reply:
Ah. I see that now. 👍